net-libs/libwebsockets/libwebsockets-1.7.6.ebuild

```diff
@@ -13,11 +13,17 @@ SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.
 LICENSE="LGPL-2.1"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="+ssl +zlib static-libs +http2 ipv6 +server -client"
+IUSE="+ssl +zlib static-libs +http2 ipv6 +server client libressl libev libuv"
```

Comment:
`IUSE` should ideally be lexically sorted for readability (though this is optional).